docs/virtual_machines/startup_scripts.md

```diff
@@ -36,9 +36,8 @@ installation, setup, and custom software provisioning.
 The general flow is:
 
 1. Seal the vm image with the Sysprep tool, for example by running:
-    ```console
-    %WINDIR%\system32\sysprep\sysprep.exe /generalize /shutdown /oobe /mode:vm
-    ```
+
+        %WINDIR%\system32\sysprep\sysprep.exe /generalize /shutdown /oobe /mode:vm
```

Comment:
I will accept the new line at L39.

re L40 ... We can't accept codeblocks out side of backtick fences

Reply:
It doesn't render properly if you use the backtick fences under a numerical bullet.

Here's how it looks today:

-----

![Screen Shot 2021-03-08 at 8 58 18 PM](https://user-images.githubusercontent.com/617316/110406941-05074180-8051-11eb-82d2-5a67a9731b92.png)

----

If you use the indent form of the code block (vs. the backtick fence form), it renders like this:

-----

![Screen Shot 2021-03-08 at 8 59 23 PM](https://user-images.githubusercontent.com/617316/110407028-2cf6a500-8051-11eb-9e08-5535acb1a12d.png)

Reply:
Right.  That's why we just need a empty line at L39

Reply:
The blank line is not enough -- if you put in a blank line and keep the backtick fence, it still renders incorrectly:

--------

![Screen Shot 2021-03-09 at 6 54 37 AM](https://user-images.githubusercontent.com/617316/110466899-55f45580-80a4-11eb-81f5-b82a73fe3ff2.png)

-----

That's why the non-backtick-fence / indent form is necessary.

Reply:
BTW, the issue is the same for the `lspci` example in `docs/virtual_machines/host-devices.md`.

This is a quirk in the rendering of code blocks inside enumerated or bulleted lists.

Reply:
We just went through a the docsify -> mkdocs migration two weeks ago.  All the headers, UTF-8 -> ascii chars changed, trailing whitespaces need to be fixed.  

I wasn't aware of the backtick problem.  It sure looks like we're going to have to visit every page again and fix.  I kinda see what you're saying re `docs/virtual_machines/host-devices.md`.  Notice that page has three different backtick schemes?

We need to standardize on ...
\`\`\`console
.....
\`\`\`
^ That is the markdown standard are we need to get it to work.  Please create a commit so we can verify.  We can always back it out.  We will then be able to see how it renders in the Netlify preview in CI tests.

Reply:
Sure, I can do that so you can see the Netify preview.  The pictures I've been pasting here have been from running `make build` locally.

FWIW, here's what it rendered as (locally for me) when I:

* added the newline
* outdented the backtick fence+`console` token:

-----
![Screen Shot 2021-03-09 at 10 51 58 AM](https://user-images.githubusercontent.com/617316/110498751-97e1c380-80c5-11eb-8f3d-132f5722986d.png)
-----

I.e.:

* The code block is outdented
* The text following it is outdented
* The numbering on the next bullet is wrong (i.e., it restarts at 1)

I think that this is because the Markdown renderer sees that the outdented backtick fence and thinks that you have ended the enumerated list.

The only way to preserve the current enumerated list is to keep indenting until the next enumerated bullet.  But when you do that, the backtick fence code block doesn't render properly.  The only way to get an indented code block to render properly (that I found, at least) is to use the indent form of the code block (vs. the backtick fence form).  Put simply: you have to "double indent" - once for the enumerated list, and again for the code block.  ...but then I don't know how you specify the `console` syntax highlight token.  🤷‍♂️

Reply:
One caveat to what I just posted (above) -- the picture shows what happens when:

* You outdent the backtick fence code block, AND
* You remove one space before the "More information..." text.

If you don't remove one space before the "More information..." text, then "More information..." gets treated as an indent-form code block (because it's indented 4 spaces).

Reply:
`make build` should absolutely NOT be used.  It will affect all the other features such as `make run`, `make check_links`, `make check_spelling`.  If you run `make` or `make help` you should see this warning....
```
  build               	 Build site. This target should only be used by Netlify and Prow
```
If you have run `make build` you will need to `sudo rm -rf ./site` or `su bash; rm -rf ./'site`.  The problem is the container engine will change ownership of files inside the container. 


I also see that behavior in Netlify.  How about this compromise... Change the diff back to what works and I'll +1.  We'll put standardizing backticks back on the top shelf to collect dust for awhile.  We can trigger an issue and deal with it later.  All for the sake of moving on from this pull req.

Reply:
> `make build` should absolutely NOT be used. It will affect all the other features such as `make run`, `make check_links`, `make check_spelling`. If you run `make` or `make help` you should see this warning....
> 
> ```
>   build               	 Build site. This target should only be used by Netlify and Prow
> ```

FWIW, I did see that warning, but I didn't see any obvious/easy way to build (that being said, since I'm an outsider and was just trying to make a small/surgical fix, I didn't look too deeply -- I saw the Sphinx-like Makefile, and `make build` resulted in something that I could preview in my browser, so I considered that "good enough" to make a PR).

> If you have run `make build` you will need to `sudo rm -rf ./site` or `su bash; rm -rf ./'site`. The problem is the container engine will change ownership of files inside the container.

Gotcha.

> I also see that behavior in Netlify. How about this compromise... Change the diff back to what works and I'll +1. We'll put standardizing backticks back on the top shelf to collect dust for awhile. We can trigger an issue and deal with it later. All for the sake of moving on from this pull req.

Cool.  Will do!